merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/ResponseSerializers.java

```diff
@@ -240,6 +239,15 @@ public static JsonValue serializeResourceSamples(final Map<String, List<Pair<Dur
         .build();
   }
 
+  public static JsonValue serializeConstraintViolations(final Map<String, List<Violation>> violations) {
+    return Json
+      .createObjectBuilder()
+      .add("constraintViolations", serializeMap(v ->
+          serializeIterable(ResponseSerializers::serializeConstraintViolation, v),
+          violations))
```

Comment:
It's a little hard to see from the formatting that the `v ->` on the first line only scopes the second line and not the third line. Maybe shift the `v ->` down a line so the whole lambda is in one line?

Reply:
Totally agree, change incoming!

Reply:
Change has been made.